storage/blockdevice/tests/UNITTESTS/SFDP/test_sfdp.cpp

```diff
@@ -254,6 +234,26 @@ TEST_F(TestSFDP, TestSFDPReadFailure)
  */
 TEST_F(TestSFDP, TestMoreRegionsThanSupported)
 {
+    /**
+     * Based on Cypress S25FS512S, modified to have one descriptor,
+     * twelve regions for test purpose.
+     */
+    const uint8_t sector_map_single_descriptor_twelve_regions[] {
```

Comment:
Can this be `constexpr`?

Reply:
I initially thought `constexpr` was for compile-time evaluation, and couldn't think of how this particular test case would benefit from it in practice, as it's just passed around like an array/pointer?

Reply:
Plausibly it won't benefit us in this case. Yes, `constexpr` means the array is treated as a constant expression which is evaluated at compile-time, which is slightly different than `const` (which means it's read-only). If we were passing it to other objects that could possibly be `constexpr` we would see some performance gains, I think. This is because the operations on the array could happen at compile time, and the compiler would just do a direct substitution of the result instead of generating the lookup/usage code. Given the way our current code is written I don't think we will see those benefits; we use pointer indirection everywhere so we'll pay the cost of de-referencing the pointers at run time, also almost none of the calling code is `constexpr` so we'd probably need to rewrite it to be `constexpr` where possible to see any benefit.

Reply:
I'm leaving it as `const` for now, because this array emulates raw bytes that are normally fetched from a device at run time.